packages/admin/admin/src/preview/CustomAction.tsx

```diff
@@ -0,0 +1,9 @@
+import { IconButton, styled } from "@mui/material";
+
+export const CustomAction: React.FC = styled(IconButton)`
```

Comment:
This name is too generic. Better would be `PreviewToolbarButton` or similar. Also, it should be in `@comet/cms-admin`, where the preview is also located.